lib/un.rb

```diff
@@ -346,6 +346,21 @@ def httpd
     end
     options[:Port] ||= 8080     # HTTP Alternate
     options[:DocumentRoot] = argv.shift || '.'
+    s = nil
+    options[:StartCallback] = proc {
+      logger = s.logger
+      logger.info("To access this server, open this file in a browser:")
```

Comment:
`file` => `url` or `link`, I wouldn't think of URLs as files.

Other than that :+1: